src/Uno.UWP/Buffers/LinearArrayPool.cs

```diff
@@ -0,0 +1,224 @@
+#nullable enable
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Windows.System;
+
+namespace Uno.Buffers
+{
+	internal class LinearArrayPool<T>
```

Comment:
Let's add tests like this one: https://github.com/unoplatform/uno/blob/78790e74a14caefa81175d7a8f5a8039f1724a4d/src/Uno.UI.Tests/Buffers/Given_ArrayPool.cs#L15